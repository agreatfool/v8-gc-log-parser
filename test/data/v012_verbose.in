[87120] Memory allocator,   used:  40420 KB, available: 1458716 KB
[87120] New space,          used:    532 KB, available:   1515 KB, committed:   4096 KB
[87120] Old pointers,       used:    972 KB, available:      0 KB, committed:   1903 KB
[87120] Old data space,     used:    801 KB, available:      1 KB, committed:   1199 KB
[87120] Code space,         used:    558 KB, available:      0 KB, committed:    996 KB
[87120] Map space,          used:    118 KB, available:      0 KB, committed:    128 KB
[87120] Cell space,         used:      4 KB, available:      0 KB, committed:    128 KB
[87120] PropertyCell space, used:     26 KB, available:      0 KB, committed:     64 KB
[87120] Large object space, used:      0 KB, available: 1457675 KB, committed:      0 KB
[87120] All spaces,         used:   3013 KB, available:   1517 KB, committed:   8515 KB
[87120] External memory reported:      8 KB
[87120] Total time spent in GC  : 1.8 ms
