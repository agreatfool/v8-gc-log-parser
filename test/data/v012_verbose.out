[
  {
    "pid": 87120,
    "version": "v0.12.x",
    "type": "verbose",
    "data": {
      "memory_allocator": {
        "used": 40420,
        "available": 1458716
      },
      "new_space": {
        "used": 532,
        "available": 1515,
        "committed": 4096
      },
      "old_pointers": {
        "used": 972,
        "available": 0,
        "committed": 1903
      },
      "old_data_space": {
        "used": 801,
        "available": 1,
        "committed": 1199
      },
      "code_space": {
        "used": 558,
        "available": 0,
        "committed": 996
      },
      "map_space": {
        "used": 118,
        "available": 0,
        "committed": 128
      },
      "cell_space": {
        "used": 4,
        "available": 0,
        "committed": 128
      },
      "property_cell_space": {
        "used": 26,
        "available": 0,
        "committed": 64
      },
      "large_object_space": {
        "used": 0,
        "available": 1457675,
        "committed": 0
      },
      "all_spaces": {
        "used": 3013,
        "available": 1517,
        "committed": 8515
      },
      "external_memory_reported": 8,
      "total_time_spent_in_gc": 1.8
    }
  }
]